slack_sdk/web/async_client.py

```diff
@@ -3191,6 +3192,75 @@ async def conversations_replies(
         )
         return await self.api_call("conversations.replies", http_verb="GET", params=kwargs)
 
+    async def conversations_requestSharedInvite_approve(
+        self,
+        *,
+        invite_id: str,
+        channel_id: Optional[str] = None,
+        is_external_limited: Optional[str] = None,
+        message: Optional[Dict[str, Any]] = None,
```

Comment:
one note about `message` here is that _if_ it is provided, `text` inside of `message` is required. not sure if that is possible to model in Python but figured I'd mention it

Reply:
No way to describe such in Python's type hints. The only approaches we can think of would be 1) accept both dict and custom class that has "text" property, 2) add runtime validation that checks if the key is not missing. That said, as long as the server-side returns a clear error code/message for the pattern, I don't think we should do extra on the client code side.